components/layout.js

```diff
@@ -1,56 +1,94 @@
-import Link from 'next/link'
-import { useState, useEffect } from 'react'
-import { slide as Menu } from 'react-burger-menu'
+import Link from "next/link";
```

Comment:
To be coherent with the linting with don't use semilicon and the imports are with a single quote.